srai/regionizers/voronoi_regionizer.py

```diff
@@ -12,8 +12,11 @@
 
 from srai.utils._optional import import_optional_dependencies
 
+from ..utils.constants import CRS
```

Comment:
I know it's a valid import but I'm more keen on using a full import in this situation.
```suggestion
from srai.utils.constants import CRS
```